src/contactDesktop/Controller.java

```diff
@@ -0,0 +1,80 @@
+package contactDesktop;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextField;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class Controller implements Initializable{
+
+    ObservableList<Contact> contacts = FXCollections.observableArrayList();
+
+    @FXML
+    TextField name;
+
+    @FXML
+    TextField phone;
+
+    @FXML
+    TextField email;
+
+    @FXML
+    ListView list;
+
+    /**
+     * Method to add a new contact. If any contact fields are not filled, an alert will pop up and wait
+     * for user input.
+     */
+    public void addContact(){
+        String contactName = name.getText();
+        String contactPhone = phone.getText();
+        String contactEmail = email.getText();
+
+        Contact contact = new Contact(contactName, contactPhone, contactEmail);
+
+        if (!name.getText().isEmpty() && !phone.getText().isEmpty() && !email.getText().isEmpty()) {
```

Comment:
One way to make this a little more efficient is to create a `isValid` method on the `Contact` object and then you could simply call `if(contact.isValid())`.  It also keeps the logic in the `Contact` object itself so that every bit of code that needs this functionality is using the same standards.